doc/api/vm.md

```diff
@@ -866,6 +866,36 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.compileFunction
+<!-- YAML
+added: REPLACEME
+-->
+* `code` {string} The JavaScript code to compile and run.
+* `params` {Array|undefined} An array of string containing all paramters for
+  the function.
+* `options` {Object}
+  * `filename` {string} Specifies the filename used in stack traces produced
+    by this script.
+    **Default:** `''`
+  * `lineOffset` {number} Specifies the line number offset that is displayed
+    in stack traces produced by this script.
+    **Default:** `0`
+  * `columnOffset` {number} Specifies the column number offset that is displayed
+    in stack traces produced by this script.
+    **Default:** `0`
+  * `cachedData` {Buffer|undefined} Provides an optional `Buffer` with V8's code
+    cache data for the supplied source.
+  * `produceCachedData` {boolean} Specifies wether to produce new cache data or
+    not.
+    **Default:** false
+  * `parsingContext` {Context|undefined} The sandbox/context in which the said
```

Comment:
The `undefined` seems unneeded if we have a default value?

Reply:
We have no `Context` type in our `tools/doc/type-parser.js`. You can add one between these lines:

https://github.com/nodejs/node/blob/71ca076ca74b4e371c74c72d9556352adee654f6/tools/doc/type-parser.js#L119-L121

Reply:
It seems like we traditionally just used `{Object}` for the type of a sandbox. That could be used instead as well.